lib/migration-stream.js

```diff
@@ -63,6 +66,10 @@ function MigrationStream() {
     var v2 = info.layers.every(function(layer) {
         return layer.version === 2;
     });
+    
+    var randLetter = String.fromCharCode(65 + Math.floor(Math.random() * 26));
```

Comment:
@millzpaugh what's this unique ID used for?

Reply:
We have to pass this into Segment since we don't have a `user_id`.